waspc/data/Generator/templates/server/src/config.js

```diff
@@ -16,29 +16,42 @@ const config = {
     port: parseInt(process.env.PORT) || 3001,
```

Comment:
Development config has origin `*` which allows all origins. 

Production configs uses the `WASP_WEB_CLIENT_URL` env variable like before.

Reply:
Is there a reason to prefer `*` over a specific URL for local dev? My only fear is if they do something in dev they think will work but will fail in prod for CORS reasons.

Reply:
Ah wait, nvmd 🤦🏻‍♂️ we **need** that for Replit sorry lol